### PR TITLE
Fix/delft3dgt 877 stopped state

### PR DESCRIPTION
**Pull request by CindyvdVries**
…h request

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -163,17 +163,18 @@ export default new Vuex.Store({
...
       _.each(this.state.models, (model) => {
         var container = _.find(this.state.modelContainers, ['id', model.id])
+        let statusLevel = 'info'
+
+        if (model.state === 'Finished') {
+          statusLevel = 'success'
+        } else if (model.state === 'Idle: waiting for user input') {
+          statusLevel = 'warning'
+        } else if (model.state === 'Running simulation') {
+          statusLevel = 'striped active'
```
Comment (SiggyF) [inline src/store/index.js:175]: striped active? What  does that mean?

File: src/store/index.js [hunk 2]
```diff
@@ -163,17 +163,18 @@ export default new Vuex.Store({
...
+        } else if (model.state === 'Stopped') {
+          statusLevel = 'secondary'
```
Comment (SiggyF) [inline src/store/index.js:177]: stopped -> secondary? I  don't see  the relation.

**Review by SiggyF — APPROVED** [review]
Some comments might be  helpful?

File: src/store/index.js (hunk 1 above)
Comment (CindyvdVries) [reply src/store/index.js:175]: bootstrap css class

**Pull request merged**